packages/manager/modules/exchange/src/account/update/update.routing.js

```diff
@@ -0,0 +1,33 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('exchange.dashboard.account.update', {
+    url: '/update',
+    component: 'exchangeAccountUpdate',
+    params: {
+      emailAccount: null,
+    },
+    redirectTo: (transition) =>
+      Promise.all([transition.injector().getAsync('emailAccount')]).then(
```

Comment:
`Promise.all` seems unecessary here

Reply:
Yep, but it easy for team to add/remove a new things comparing if we implement the single call promise.